pkg/storage/image.go

```diff
@@ -616,6 +616,8 @@ func splitDockerDomain(name string) (domain, remainder string) {
 	return
 }
 
+// Resolve names resolves an image name into a storage image ID or a fully-qualified image name (domain/repo/image:tag)
+// Will return an empty array if err != nil
```

Comment:
"... a slice of candidate fully-qualified image names or IDs" might be more accurate.

Reply:
“Will only return an empty slice if err != nil.”

I'd like an ”only” in there.  I'm agnostic on @nalind's “of candidate … IDs” bit.

Also, these should be full sentences ending in periods, since [they'll show up in the godoc][1].  We'll want to update [this entry as well][2] to keep it in sync.

[1]: https://godoc.org/github.com/kubernetes-incubator/cri-o/pkg/storage
[2]: https://github.com/kubernetes-incubator/cri-o/blob/0b736bb43fb9f6ffa29b477be9e292bec3686ef4/pkg/storage/image.go#L98-L100

Reply:
Updated comment.